docker/docker+.bsh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ $# == 0 ]]; then
+  FILES=(-)
+else
+  FILES=("${@}")
+fi
+
+CUR_DIR=$(basename ${BASH_SOURCE[0]})
```

Comment:
Should this be `dirname` instead? Otherwise this is just `docker+.bsh` when run on my mac.